Assets/_BForBoss/_Core/Scripts/Environment/EnemySpawnAreaBehaviour.cs

```diff
@@ -84,8 +86,16 @@ private void SpawnEnemy()
         {
             Debug.Log("<color=red>Spawn Enemy</color>");
             var enemy = _enemyContainer.GetEnemy();
+            
             enemy.OnRelease += HandleOnEnemyReleased;
             enemy.transform.SetPositionAndRotation(transform.position, transform.rotation);
+            enemy.transform.SetParent(transform, true);
+            
+            //https://answers.unity.com/questions/771908/navmesh-issue-with-spawning-players.html
+            NavMeshAgent agent = enemy.GetComponent<NavMeshAgent>();
+            agent.enabled = false;
+            agent.enabled = true;
```

Comment:
can u not do something like this within the `AgentNavigationBehaviour`? 

Or create a `Spawn` State in the AI?

Reply:
I tested quite a bit with enabling and disabling the agent in `AgentNavigationBehaviour` but the results seem inconsistent or hard to follow!

It looks like only after we Set the transform of the Instantiated EnemySpawn do we have to disable/enable the agent for the most consistent results